source/code-generation/query-converter/how-query-converter-uses-your-data.txt

```diff
@@ -0,0 +1,52 @@
+.. _rm-query-converter-data-usage:
+
+==================================
+How Query Converter Uses Your Data
+==================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Query Converter currently uses the `Azure OpenAI Service
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+hosted by Microsoft. 
+
+When you use Query Converter, the following information is sent to 
+MongoDB's backend and the third party AI provider:
+
+- The full text of your query or stored procedure.
+- The definition of any relational database tables used in the query.
+  These definitions include the table names, column names, data types,
+  and any primary or foreign keys relationships.
+- The definition of your MongoDB collections and mapping rules. 
+  These definitions include the relationships between the collections 
+  and tables used in the query and the collection names, field names 
+  and data types.
+
+The information that is sent is not shared with any other third 
+parties or stored by the AI provider. Relational Migrator does not send 
+database connection strings, credentials, or any raw data from your 
+databases.
+
+By default, your original query text and the AI-converted query 
+telemetry data is stored by MongoDB for up to one year to help us 
+provide support to customers and improve the service. If you do not want
+your data to be stored, you have the option to opt out:
+
+- You can disable the :guilabel:`Allow queries to be stored temporarily` 
+  toggle switch in global project settings.
+- You can uncheck the :guilabel:`Allow MongoDB to temporarily store the 
+  source and converted queries to import the product feature` setting 
+  when you enable query converter.
+
+Disable Query Converter
```

Comment:
[Question/Suggestion]

So technically we can't properly disable this feature but we can simply not use it/enable it? If so, maybe we can make this heading the "Details" heading from concept page prototype. I think with the current title it does come across like you can turn it on and off.